app/Http/Requests/Admin/QuestionRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Requests\Admin;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class QuestionRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, mixed>
+     */
+    public function rules()
+    {
+        return [
+            'content' => ['required',Rule::unique('questions')->ignore($this->question)],
+            'category_id' => ['required'],
+            'answer' => ['required','array'],
+            'radio-answer' => ['required','in:0,1,2,3'],
```

Comment:
radio-answer sao lại validate in:0,1,2,3
Sau này khi edit question value của các answer nó là id tương ứng trong db đó

Reply:
![image](https://user-images.githubusercontent.com/59141451/185281702-7b21f53e-3e92-461b-9963-e0b95d235289.png)
Cái ô tích chọn đáp án e để set value từ 0 đến 3, nên e tính validate như thế, chỉ chấp nhận những giá trị đó, mà e nghĩ vẫn bị fake được input radio, để e xử lý lại a ạ